starknet_py/tests/e2e/fixtures/utils.py

```diff
@@ -0,0 +1,149 @@
+# pylint: disable=redefined-outer-name
+
+import asyncio
+import json
+import subprocess
+import time
+from contextlib import closing
+import socket
+from typing import Generator
+
+import pytest
+import pytest_asyncio
+
+from starknet_py.contract import Contract
+from starknet_py.net import AccountClient
+from starknet_py.tests.e2e.fixtures.constants import (
+    TYPED_DATA_DIR,
+    CONTRACTS_DIR,
+)
+from starknet_py.utils.data_transformer.data_transformer import CairoSerializer
+from starknet_py.utils.typed_data import TypedData
+
+
+# This fixture was added to enable using async fixtures
+@pytest.fixture(scope="module")
+def event_loop():
+    policy = asyncio.get_event_loop_policy()
+    loop = policy.new_event_loop()
+    yield loop
+    loop.close()
```

Comment:
Maybe let's have it in a separate file like `event_loop` just to emphasise we're modifying this.